src/mlpack/core/tree/vantage_point_tree/typedef.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file typedef.hpp
+ *
+ * Template typedefs for the VantagePointTree class that satisfy the
+ * requirements of the TreeType policy class.
+ */
+#ifndef MLPACK_CORE_TREE_VANTAGE_POINT_TREE_TYPEDEF_HPP
+#define MLPACK_CORE_TREE_VANTAGE_POINT_TREE_TYPEDEF_HPP
+
+// In case it hasn't been included yet.
+#include "../vantage_point_tree.hpp"
+
+namespace mlpack {
+namespace tree {
+  
```

Comment:
We should add a quick comment here on what the tree is, so that users will find it in the documentation: http://mlpack.org/docs/mlpack-git/doxygen.php?doc=namespacemlpack_1_1tree.html#typedef-members (I wish that page was less ugly but doxygen is not the best).